crates/primitives/starknet/src/execution/felt252_wrapper.rs

```diff
@@ -109,13 +109,35 @@ impl TryFrom<&[u8]> for Felt252Wrapper {
     }
 }
 
-/// [`Felt252Wrapper`] from u128.
-impl From<u128> for Felt252Wrapper {
-    fn from(value: u128) -> Self {
-        Felt252Wrapper::try_from(U256::from(value)).unwrap()
-    }
+macro_rules! impl_from_primitive {
+    ($($t:ty),*) => {
+        $(
+            impl From<$t> for Felt252Wrapper {
+                fn from(value: $t) -> Self {
+                    Felt252Wrapper::try_from(U256::from(value)).unwrap()
```

Comment:
shouldn't we use instead `FieldElement::from` to avoid creating a useless (too big) `U256` and then `unwrap`?